.travis.yml

```diff
@@ -12,7 +12,7 @@ cache:
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
+    - php: 7.1.18
```

Comment:
why locked at minor version?

Reply:
also update required version in composer.json or add the one from composer here too

Reply:
@JarJak it's not about _our_ required version. it's about a version that's borken in Travis specifically. I've added a note to it.